UnleashClient/__init__.py

```diff
@@ -157,6 +161,11 @@ def is_enabled(self,
         :param default_value: Allows override of default value.
         :return: True/False
         """
+        if "appName" in context.keys():
```

Comment:
would it make sense to drag these out as a separate function?

Reply:
I would recommend something like:

```Python
default_values = {
    "appName": self.unleash_app_name,
    "environment": self.unleash_environment
}

context.update(default_values)
```

This will always replace the "appName" and "environment" keys in the context.